lua/rns.lua

```diff
@@ -3,7 +3,7 @@ dofile("rns-lib.lua")
 
 -- echo whatever is received
 
--- this function specifies the mutalation to the packet
+-- this function specifies the mutatation to the packet
```

Comment:
not certain

Reply:
In the accompanied `rns-specs` file packet mangling is described as:

>  - distort the compression pointers in the final packet
>  - swap bytes in the final packet
>  - swap bits in the final packet

So perhaps we should change the sentence into:

`-- this function disfigures the packet`

What do you think?

Reply:
I'd be half tempted to just go with `manipulation of`.